### ISSUE TITLE
Publish packages

### ISSUE THREAD

**Issue by rugk** [rabite0/hunter#2]
It would be great if you could publish packages/let people contribute packages.

So you e.g. get a list like this: https://github.com/BurntSushi/ripgrep/tree/31adff6f3c4bfefc9e77df40871f2989443e6827#installation
See also: https://rust-lang-nursery.github.io/cli-wg/tutorial/packaging.html

**Comment by rabite0** [comment]
Running cargo --publish right now, but it sure takes forever.

Or do you mean binary packages? I might look into that, depending on how much work it is. I'd also be happy to accept contributions.

**Comment by rabite0** [comment]
So, yeah, I did take a look and I don't really want to get into that at this point. Maybe a generic tar.gz at most. But if someone wants to provide distribution specific binary packages, I'm willing to accept them. Will close this issue at some point if no one steps up.

**Comment by rugk** [comment]
Just leave it open though. It certainly takes time until this becomes popular, people step up to package it for their favorite distro and last, but not least, it is unlikely one can do it for a lot of distros. It will certainly be one distro at a time…

After all, it is also [the last thing described in the docs](https://rust-lang-nursery.github.io/cli-wg/tutorial/packaging.html#getting-your-app-into-package-repositories). It's obvious, this takes time…

**Comment by m040601** [comment]
Another vote for releasing it as a binary on github.
This is a usefull generall use tool, catering fo the average user.
Many already have ranger or similar clones installed.
Why would anyone want to check why "hunter" may be better/different, if you dont make it dead easy to do that ?

Not everyone is a developer, has the entire rust distribution installed or as a simple user is familiar with compiling.
Besides that releasing a binary, would make it easier for other third parties  to create a package for a Linux distro, based on the binary (ex. Ubuntu, Archlinux AUR). (Dont' confuse a package based on the released binary with  a package built from the source, that's another thing, https://aur.archlinux.org/packages/hunter/)

**Comment by rabite0** [comment]
Thanks to @straussdd there are binary packages now.

**Comment by dufferzafar** [comment]
@rabite0 Where can I download binary packages from? They're not visible in GitHub releases, like for other rust tools.

**Comment by dufferzafar** [comment]
I looked at Travis CI and all the builds seem to be failing: https://travis-ci.com/github/rabite0/hunter/branches

**Comment by rabite0** [comment]
Yeah, they've been failing since I added video previews, since the build system doesn't handle
